flight/Engine.php

```diff
@@ -308,24 +300,27 @@ public function get(?string $key = null)
     /**
      * Sets a variable.
      *
-     * @param mixed      $key   Key
-     * @param mixed|null $value Value
+     * @param string|iterable<string, mixed> $key
+     * Variable name as `string` or an iterable of `'varName' => $varValue`
+     * @param mixed $value Ignored if `$key` is an `iterable`
      */
     public function set($key, $value = null): void
     {
-        if (\is_array($key) || \is_object($key)) {
+        if (\is_iterable($key)) {
             foreach ($key as $k => $v) {
                 $this->vars[$k] = $v;
             }
-        } else {
-            $this->vars[$key] = $value;
+
+            return;
```

Comment:
I think this is ok. This seems to be a better way of handling it than it did before....I'll just need to keep it in mind for the unit tests.

Reply:
The cyclomatic complexity is still 2, there is still an implicit if else, so testing both branches would give 100% coverage